### PR TITLE
Limit image file size while copying into editor

### LINKED ISSUE #418: Max image file size bug
Ref: https://github.com/bigbinary/neeto-kb-web/issues/2574

### PR DESCRIPTION
**Pull request by gaagul**
Fixes #418 

Description

- Added file size restriction when pasting an image into the editor.

Reviewers

@AbhayVAshokan _a

### CODE REVIEW

File: lib/components/Editor/CustomExtensions/Image/ExtensionConfig.js [hunk 1]
```diff
@@ -130,10 +133,15 @@ const ImageExtension = Node.create({
...
 const upload = async (file, url) => {
-  const uploader = new DirectUpload({ file, url });
-  const { blob_url: imageUrl } = await uploader.create();
+  if (file.size <= MAX_IMAGE_SIZE) {
+    const uploader = new DirectUpload({ file, url });
+    const { blob_url: imageUrl } = await uploader.create();
+
+    return imageUrl;
+  }
+  Toastr.error(`Image size should be less than ${MAX_IMAGE_SIZE / 1024} KB`);
```
Comment (AbhayVAshokan) [inline lib/components/Editor/CustomExtensions/Image/ExtensionConfig.js:142]:
```suggestion

  const imageSizeInMB = MAX_IMAGE_SIZE / (1024 * 1024);
  Toastr.error(`Image size should be less than ${imageSizeInMB} KB`);
```

Showing the error message in MB is more readable and better in terms of UX.

**Review by AbhayVAshokan — COMMENTED** [review]
@gaagul _a why is the `bug` label added to the PR? As I mentioned earlier, it should be either `patch`, `minor` or `major`. Keep that in mind for your next PR.

**Comment by gaagul** [comment]
@AbhayVAshokan _a

**Review by AbhayVAshokan — APPROVED** [review]
@gaagul this change would be a `patch` instead of a `minor` change. This does not introduce a breaking change or introduces a new feature. This added an enhancement that was necessary for the image upload functionality during pasting an image.

Ref: https://semver.org/

**Pull request merged**